docs/backend-api/README.md

```diff
@@ -13,82 +13,122 @@ Inside a _tenant namespace_ the client creates `pipelinerun` resources for each
+
+- Service account `<client_namespace>::default` (where `<client_namespace>` is the namespace where the `Tenant` resource belongs to) has the permissions needed to manage further resources in the tenant namespace.
+
+The Steward controller periodically checks the actual state of the resource and tries to change it to the desired state:
+
+- The role binding in the tenant namespace gets updated/recreated if needed, for instance if the client namespace's annotation `steward.sap.com/tenant-role` (defining the RBAC role to be assigned to the above-mentioned service accounts) has changed.
+
+- If `status.tenantNamespaceName` refers to a namespace that does not exist anymore, the ready condition is set to `False` indicating that the tenant is no longer ready to be used. As this never happens under normal circumstances and probably means that data has been lost, the tenant namespace will not be recreated automatically. Operators should monitor tenants, and must  analyze and fix the underlying issue if such situations occur.
+
+
+### Examples
 
-```bash
-$ kubectl -n <steward-client1> get tenant <tenantId> -oyaml
-```
-_(shortened example yaml)_
 ```yaml
+apiVersion: steward.sap.com/v1apha1
+kind: Tenant
+metadata:
+  name: tenant1
+  namespace: steward-c-client1
 status:
-  message: Tenant namespace successfully prepared
-  progress: Finished
-  result: success
-  tenantNamespaceName: stu-tn-cl1-test-tenant-09a530
+  conditions:
+  - type: Ready
+    status: "True"
+    lastTransitionTime: "2019-11-01T08:15:36Z"
+  tenantNamespaceName: steward-t-client1-tenant1-83a4cf
 ```
 
-| Parameter | Description |
+
+#### Fields
+
+| Field | Description |
 | --------- | ----------- |
-|`status.message` | A message describing the latest status |
-|`status.progress` | The current progress of processing the tenant resource **(deprecated)**. Possible values:<br>`['', 'InProcess', 'CreateNamespace', 'GetServiceAccount', 'AddRoleBinding', 'Finalize', 'Finished']` |
-|`status.result` | The result of the resource processing. Possible values:<br>`['', 'success', 'error_infra', 'error_content']` |
-|`status.tenantNamespaceName` | The name of the namespace to be used for this tenant |
+| `status.conditions` | (array) A list of condition objects describing the lastest observed state of the resource. For each type of condition at most one entry exists. See _Conditions_ below. |
+| `status.conditions[*].type` | (string) The type of the condition. See _Conditions_ below. |
+| `status.conditions[*].status` | (string,optional) The status of the condition with one of the values `True`, `False` and `Unknown`. A condition that is not listed in `status.conditions` has status `Unknown`. |
+| `status.conditions[*].reason` | (string,optional) A unique, one-word, camel-case reason for the condition's last transition. |
```

Comment:
Do we have a defined set of possible reasons? It should be listed either here or in the `Conditions` section.
If the reasons are not defined what is the value, compared to message? It's some kind of short-message then.

Reply:
9e59518d068877c457b5a7b64dc9d9ea0a1db5b9 adds all this in the Conditions section.